### PR TITLE
Completed footer component

### PR DESCRIPTION
**Pull request by Jayas121**
Created Footer and added about page

### CODE REVIEW

File: components/Footer/index.tsx [hunk 1]
```diff
@@ -0,0 +1,148 @@
+
+// import Link from "next/link";
+import Image from "next/image";
+// import { useState } from "react";
```
Comment (gitgenjutsu) [inline components/Footer/index.tsx:4]: Please removed commented code, if no use

File: components/Footer/index.tsx [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+
+import { ReactElement } from 'react';
+
+
```
Comment (gitgenjutsu) [inline components/Footer/index.tsx:8]: Do indentation of code

File: pages/_app.tsx [hunk 3]
```diff
@@ -1,6 +1,14 @@
...
 import '../styles/globals.css'
 import type { AppProps } from 'next/app'
+// import '../styles/header.css'
+
+
 
 export default function MyApp({ Component, pageProps }: AppProps) {
-    return <Component {...pageProps} />
+    return(
+        <Layout>
```
Comment (gitgenjutsu) [inline pages/_app.tsx:10]: Do indentation in all files

File: styles/header.css [hunk 4]
```diff
@@ -0,0 +1,88 @@
+/*.More{
```
Comment (gitgenjutsu) [inline styles/header.css:1]: Removed unused code from all files, if not needed

**Review by gitgenjutsu — CHANGES_REQUESTED** [review]
Please solve conflicts and comments

**Comment by Jayas121** [comment]
updated footer

**Comment by Jayas121** [comment]
updated Footer

**Pull request merged**